pages/privacy-policy.js

```diff
@@ -15,7 +15,7 @@ const PrivacyPolicy = () => {
         <section className="pt-4 pt-md-11 home-background">
           <div className="container mx-auto">
             <div className="row align-items-center justify-content-center mb-4 bg-image-policy-header">
-              <h2 className="h2 text-center">Privacy Policy</h2>
+              <h1 className="h2 text-center">Amplication Privacy Policy</h1>
```

Comment:
you use h1 but the class name is `h2 text-center`

Reply:
I didnt want to chang the style, but still for semantic HTML and SEO we need to use H1